js/select4.js

```diff
@@ -122,7 +124,7 @@
         var optionLi = this.optionsUl.childNodes[i];
         optionLi.className = '';
       }
-      this.optionsUl.childNodes[idx].className = 'selected';
+      this.optionsUl.childNodes[idx-1].className = 'selected';
```

Comment:
What issue are you solving with this? I don't understand what do you gain by selectng the previous child